subsystem/host_v2_test.go

```diff
@@ -222,3 +233,59 @@ func updateHostV2(ctx context.Context, request *installer.V2UpdateHostParams) *m
 	Expect(response.Payload).NotTo(BeNil())
 	return response.Payload
 }
+
+func isHostInStateV2(ctx context.Context, infraEnvID strfmt.UUID, hostID strfmt.UUID, state string) (bool, string, string) {
+	rep, err := userBMClient.Installer.V2GetHost(ctx, &installer.V2GetHostParams{InfraEnvID: infraEnvID, HostID: hostID})
+	Expect(err).NotTo(HaveOccurred())
+	h := rep.GetPayload()
+	return swag.StringValue(h.Status) == state, swag.StringValue(h.Status), swag.StringValue(h.StatusInfo)
+}
+
+func waitForHostStateV2(ctx context.Context, state string, timeout time.Duration, hosts ...*models.Host) {
```

Comment:
Maybe use `Eventually` like here: https://github.com/openshift/assisted-service/blob/master/subsystem/kubeapi_test.go#L1482

Reply:
There is also `Consistently`

Reply:
fixed